backend/src/main/java/com/google/rolecall/services/LocalStorageService.java

```diff
@@ -80,6 +92,18 @@ public Resource loadAsResource(AssetType type, String filename)
     return resource;
   }
 
+  @Override
+  public String loadAsUrl(AssetType type, String filename)
+      throws InvalidParameterException, FileNotFoundException {           
+        Blob blob = storage.get(
+          BlobId.of(bucketName, String.format("%s/%s", type.location, filename)));
+        if (blob == null || !blob.exists()) {
+          throw new FileNotFoundException("No file found for " + filename + ".");
+        }
+        return storage.signUrl(blob.toBuilder().setContentType("text/plain").build(), 1, TimeUnit.DAYS, SignUrlOption.withV4Signature()).toString();
```

Comment:
Since this does not function, this should throw an unimplemented exception.

Reply:
done